src/deploy/osp_deployer/test-powerflex.py

```diff
@@ -0,0 +1,166 @@
+#!/usr/bin/env python3
+
+# Copyright (c) 2015-2020 Dell Inc. or its subsidiaries.
+#
```

Comment:
Should be file be included ?

Reply:
This looks like a sanity test. Can we move it to deployment_validation?

Reply:
Test file, will remove it.

Reply:
> Test file, will remove it.

Why not improve it and make it a sanity test for Flex deployments. We have to manually run those tests anyways